backend/medtagger/repositories/users.py

```diff
@@ -35,3 +38,15 @@ def get_user_by_email(user_email: str) -> Optional[User]:
     def get_user_by_id(user_id: int) -> User:
         """Get user with given id."""
         return User.query.filter(User.id == user_id).one()
+
+    @staticmethod
+    def set_user_info(user_id: int, firstName: str, lastName: str) -> None:
+        """Set user's info."""
+        try:
+            user = UsersRepository.get_user_by_id(user_id)
+        except NoResultFound:
+            raise InvalidArgumentsException('User with this id does not exist.')
```

Comment:
As you cannot raise an API Exception here, please do such handling in business logic (`backend/medtagger/api/users/business.py`)